src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -880,6 +880,30 @@ static void StoreNullableIntArray(HttpContext httpContext, int?[]? tryParsable)
         Assert.Null(httpContext.Items["tryParsable"]);
     }
 
+    [Fact]
+    public async Task RequestDelegateHandlesOptionalStringValuesFromNullQueryString()
+    {
+        var httpContext = CreateHttpContext();
+        httpContext.Request.Query = new QueryCollection(new Dictionary<string, StringValues>
+        {
+            ["tryParsable"] = (string?)null
+        });
+
+        static void StoreNullableIntArray(HttpContext httpContext, StringValues? tryParsable)
+        {
```

Comment:
```suggestion
        {
            Assert.False(tryParsable.HasValue);
```


Whether or not `StringValues.Empty` and `(string?)null` are functionally equivalent, `StringValues.Empty` and `(StringValues?)null` are still different values.

If I declared a parameter to be `StringValues? tryParsable` (assuming we decide to support `Nullable<StringValues>`), I'd expect `tryParsable.HasValue` to be false if the parameter is missing. After all, it's not necessary to make the parameter nullable to pass in `StringValues.Empty`.